tools/flexray2ip/Makefile

```
@@ -0,0 +1,21 @@
+CC=gcc
+CFLAGS=-I. -std=c99 -D_DEFAULT_SOURCE
+DEPS = flexray.h
+.DEFAULT_GOAL := all
+
+%.o: %.c $(DEPS)
+	$(CC) -c -o $@ $< $(CFLAGS)
+
+wice-flexray2ip: flexray2ip.o
```

would the following change make sense?
```suggestion
flexray2ip: flexray2ip.o
```